lib/ansible/modules/cloud/azure/azure_rm_storageblob.py

```diff
@@ -91,23 +90,23 @@
             - source
     state:
         description:
-            - Assert the state of a container or blob.
+            - State of a container or blob.
             - Use state C(absent) with a container value only to delete a container. Include a blob value to remove
-              a specific blob. A container will not be deleted, if it contains blobs. Use the force option to override,
+              a specific blob. A container will not be deleted, if it contains blobs. Use the I(force) option to override,
               deleting the container and all associated blobs.
             - Use state C(present) to create or update a container and upload or download a blob. If the container
               does not exist, it will be created. If it exists, it will be updated with configuration options. Provide
               a blob name and either src or dest to upload or download. Provide a src path to upload and a dest path
               to download. If a blob (uploading) or a file (downloading) already exists, it will not be overwritten
-              unless the force parameter is true.
+              unless I(force=true).
         default: present
         choices:
             - absent
             - present
     public_access:
         description:
-            - Determine a container's level of public access. By default containers are private. Can only be set at
-              time of container creation.
+            - Determine a container's level of public access. By default containers are private.
```

Comment:
Is this default a setting of the Ansible module? or is it a default on Azure that the module relies on?

```suggestion
            - A container's level of public access. By default containers are private.
```